pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -969,15 +965,19 @@ func (cm *containerManagerImpl) GetDynamicResources(pod *v1.Pod, container *v1.C
 		// a set of CDIDevices from a different kubelet plugin. In the future we may want to
 		// include the name of the kubelet plugin and/or other types of resources that are
 		// not CDIDevices (assuming the DRAmanager supports this).
```

Comment:
The TODO that includes this comment is now obsolete.

Reply:
Ack.